### ISSUE TITLE
RobotFramework needs a new logo

### ISSUE THREAD

**Issue by zephraph** [robotframework/robotframework#2100]
The toy robot has been with us for a while, but I think it's time to get a more marketable logo. 

If you're artistic and you'd like to design the new face of RF, please submit your ideas! Also, if you submit an idea please list what license the image will be released under.

**Comment by pekkaklarck** [comment]
Apart from looking great, the new logo should have these characteristics:

- Free license. Even attribution required by [CC BY](http://creativecommons.org/licenses/by/4.0/) would be inconvenient so perhaps [public domain](https://creativecommons.org/publicdomain/zero/1.0/) would be good.
- Scales down to favicon size. This version can be slightly different than the full scale one.

Person contributing the logo will be added to AUTHORS.txt file along with other contributors. Otherwise we cannot really promise a prize or compensation.

**Comment by nonline** [comment]
Any other requirements?

**Comment by nonline** [comment]
![robot_256](https://cloud.githubusercontent.com/assets/13943203/9439032/7f38d218-4a63-11e5-9f2d-b9a4f2ae2949.png)
![robot_128](https://cloud.githubusercontent.com/assets/13943203/9439033/7f39a134-4a63-11e5-98d5-5f0585289ce3.png)
![robot_64](https://cloud.githubusercontent.com/assets/13943203/9439034/7f3abc36-4a63-11e5-9dfa-9600dbad3972.png)
![robot_32](https://cloud.githubusercontent.com/assets/13943203/9439035/7f3f5e6c-4a63-11e5-90f3-e78a88e21a54.png)

**Comment by pekkaklarck** [comment]
Thanks @nonline for the first proposal here!

Does the reference to CC-BY licence mean that you'd like to use that license if we select this logo? I generally like that license, but the required attribution is a problem if we want to make the logo very widely used. Regardless the logo, I'd like to use it, for example, on slides and other docs and adding attribution everywhere would not be convenient. What do others think about the license issue?

**Comment by nonline** [comment]
If You prefer GPL or other type for license (I usually use CC) there is no problem for me. By attribution I meant just some credits on a project website, nothing special. It' can be even a Beerware ^^

**Comment by pekkaklarck** [comment]
No, I don't prefer GPL. I don't think it works well with loges or other images and I think it would require copyright information be displayed with the logo too.

The problem with CC-BY is that it requires people using the work to always show attribution. I don't think just having the attribution on the project pages and in `AUTHORS.txt` file is alone. It might be possible that the copyright holder could give more free usage rights, but then it might be better to use [public domain](https://creativecommons.org/publicdomain/zero/1.0/) in the first place. A potential problem with public domain is that anyone can use the work for anything. I'd like the logo usage to be somehow restricted to Robot Framework related topics, but I don't know is that possible legally or could we actually enforce it anyway.

This is clearly something we need to investigate a bit more. I hope we would get some more logo proposals too!

**Comment by HelioGuilherme66** [comment]
Maybe looking for an example of an open licensing, for example the Tux
(Linux penguin), it came also from a contest I think and is copyleft.

**Comment by qitaos** [comment]
@pekkaklarck I wrote a chinese book about Robotframework for chinese beginners. I want to use the robot framework logo on the book cover. The book will be published before Nov. Can I use the logo? Thank you

**Comment by pekkaklarck** [comment]
@qitaos There isn't any official logo yet. You can use the toy robot photo used in various slide decs, in Windows installer, and elsewhere if you want.

Also, great to hear about the book!

**Comment by jChapman** [comment]
I'm not much of an artist, but I gave this a shot. [You can read about my initial efforts here](http://superfluoussextant.com/robot-logo.html)

Here are [my favorites](http://superfluoussextant.com/images/bestBots.png) from what I've done so far and here are some other designs (other than what is listed on that blog post):

- http://superfluoussextant.com/images/moreBots.png
- http://superfluoussextant.com/images/evenMoreBots.png

I'd be happy to iterate on designs and/or colors if you have any feedback or suggestions.

I usually release my art under CC0, but I'd be happy to donate it to RF and then let you license it anyway you want.

**Comment by pekkaklarck** [comment]
This is awesome! First of all, thanks for the hilarious blog post. Made my fluey mood a lot better. :)

Logos at http://logo.pizza you mentioned are really nice, both good looking and simple. I especially liked simple black frames like http://logo.pizza/deer-leap. Something like that perhaps with a little red and green highlight similarly as in http://logo.pizza/supply-belt could work really well with RF.

My favorites from your designs were those pretty simple robot arms with thick border, but head might be easier to recognize. Full robot body easily gets too detailed that it won't work too well in small size.

**Comment by JimiC** [comment]
I've been looking all over to find an SVG version of the logo to add in our [vscode-icons](https://github.com/robertohuertasm/vscode-icons) collection but I think that the robots are sabotaging my efforts.
